src/id.rs

```diff
@@ -40,6 +40,12 @@ impl Id {
     }
 }
 
+impl std::fmt::Debug for Id {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        self.as_u32().fmt(f)
```

Comment:
Hmm. I think it would be better to print *something* other than just a number -- e.g., `Id(N)`. I've usually found there is value in having some more "grep-able".